lib/ui/blocs/deeplink/deeplink_bloc.dart

```diff
@@ -11,21 +11,13 @@ part 'deeplink_state.dart';
 part 'page_command.dart';
 
 class DeeplinkBloc extends Bloc<DeeplinkEvent, DeeplinkState> {

```

Comment:
Does this disable deep links?

How are deep links handled now?

Reply:
this code was never used